### PR TITLE
chore: discord and github icons

### LINKED ISSUE #222: Add discord link to website and github README
Invite link: https://join.slack.com/t/functionless/shared_invite/zt-1asl2cm2p-UGfWsH0TD~DI~6IXODEM7A

- [ ] functionless.org
- [ ] https://github.com/functionless/functionless/blob/main/README.md

Hopefully with a nice icon and badge?

### PR DESCRIPTION
**Pull request by thantos**
Closes #222

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Functionless `λ<`
 
-[![npm version](https://badge.fury.io/js/functionless.svg)](https://badge.fury.io/js/functionless)
+[![npm version](https://badge.fury.io/js/functionless.svg)](https://badge.fury.io/js/functionless) [![Slack](https://img.shields.io/badge/slack-community-brightgreen?logo=slack)](https://functionless.slack.com/signup#/domain-signup)
```
Comment (sam-goodwin) [inline README.md:3]: Does it render a badge?

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -116,8 +119,15 @@ const config = {
           { to: "/blog", label: "Blog", position: "left" },
           {
             href: "https://github.com/functionless/functionless",
-            label: "GitHub",
             position: "right",
+            className: "header-github-link",
+            "aria-label": "GitHub Repository",
+          },
+          {
+            href: "https://functionless.slack.com",
```
Comment (sam-goodwin) [inline website/docusaurus.config.js:127]: Is this an invite link?

**Review by sam-goodwin — APPROVED** [review]

File: website/docusaurus.config.js (hunk 2 above)
Comment (thantos) [reply website/docusaurus.config.js:127]: No. Just signin/signup. I think we need to turn off the email requirement. I don't like the idea of link that will stop working after x joins.

File: README.md (hunk 1 above)
Comment (thantos) [reply README.md:3]: image

File: website/docusaurus.config.js (hunk 2 above)
Comment (sam-goodwin) [reply website/docusaurus.config.js:127]: Agreed

Comment (sam-goodwin) [reply website/docusaurus.config.js:127]: By email requirement, do you mean Functionless.org

Comment (thantos) [reply website/docusaurus.config.js]: Yeah. Looks like we cannot open up the allow list. Only invite or Allowlist options.

**Comment by thantos** [comment]
image

**Comment by sam-goodwin** [comment]
Let's update this to discord. https://discord.gg/zercqjtE

**Review by sam-goodwin — APPROVED** [review]

**Pull request merged**